server/document.js

```diff
@@ -97,11 +97,14 @@ export class NextScript extends Component {
 
   render () {
```

Comment:
@bringking I'll prefer using `<script async />` even that article mentioned it's blocks.
In my tests and experience, browser will start painting the page before even the script hits the server.
(script async is not yet done)
But I'll do it.